vue/README.md

```diff
@@ -18,15 +18,10 @@ Then, use it as follows:
 // main.js
 
 import Vue from 'vue'
-import App from './app.vue'
 import textMask from 'vue-text-mask'
 
 Vue.directive('text-mask', textMask)
 
-new Vue({
-  el: 'body',
-  components: { App }
-})
```

Comment:
Is this part really not necessary to bootstrap the component? I see that you need this `new Vue` thing in the example code below?